src/internal/connector/sharepoint/data_collections.go

```diff
@@ -41,6 +41,7 @@ func DataCollections(
 		scopes      = b.DiscreteScopes(siteIDs)
 		collections = []data.Collection{}
 		errs        error
+		spcs        []data.Collection
```

Comment:
This should get moved to line ~55.  It's dangerous to share it across loop iterations.

Reply:
Resolved within [54689ae](https://github.com/alcionai/corso/pull/1788/commits/54689ae7cdc60a33cce256b5cc4fd35be12282f4)